src/components/Nav/NavItem.test.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 The Matrix.org Foundation C.I.C.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// @vitest-environment happy-dom
+
+import type { IWindow } from "happy-dom";
+import { Provider } from "jotai";
+import { useHydrateAtoms } from "jotai/utils";
+import { create } from "react-test-renderer";
```

Comment:
Hm, should we try to converge on using React Testing Library for everything? Or is that not worth it at this time?